hack/update-protos.sh

```diff
@@ -18,9 +18,17 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-if [[ $# -lt 3 ]]; then
-  echo "Usage: bazel run //hack:update-protos [-- tmpdir]" >&2
```

Comment:
perhaps we should preserve usage somewhere? we seem to still accept args to this script?

Reply:
Done